### PR TITLE
Restricted the listing to reset services in the current namespace

### PR DESCRIPTION
**Pull request by rhaschke**
Forward port of #188.

### CODE REVIEW

File: sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py [hunk 1]
```diff
@@ -39,11 +39,12 @@ def generate_reset_services_list(self):
...
         service_list = []
 
-        # We first read the list of available motor reset services
+        # We first read the list of available motor reset services in this namespace
         # this will allow us to avoid having to know the name of the robot driver node
+        ns = rospy.get_namespace()
         while not reset_service_list:
             rospy.sleep(0.5)
-            service_list = rosservice.get_service_list()
+            service_list = rosservice.get_service_list(namespace=ns)
```
Comment (biofotis) [inline sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py:47]: Its failing one test now 
max time [60.0s] allotted for test [test_hand_finder_py] of type [sr_utilities/test_hand_finder.py]
Can you see our tests by the way?

**Review by biofotis — CHANGES_REQUESTED** [review]

File: sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py (hunk 1 above)
Comment (rhaschke) [reply sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py:47]: No, unfortunately, I can't access your AWS tests. That's really a hurdle in filing clean PRs. Why don't you use open CI providers, like GitHub actions, CircleCI, Travis?
There is also a great standard CI framework for ROS packages that can be employed at most of these CI providers: https://github.com/ros-industrial/industrial_ci

Comment (biofotis) [reply sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py:47]: ok now it should work if you could try again please?

Comment (rhaschke) [reply sr_utilities/scripts/sr_utilities/calibrate_hand_finder.py:47]: Done.

**Pull request merged**